webportal/webapp/src/app/account/loginsync/services/loginsyncservice.service.ts

```diff
@@ -25,7 +24,7 @@ export class LoginSyncService {
   userAccountsLinked(): Observable<any> {
     return this.http.get(this.accountLinkedUrl, this.httpOptions)
       .map((response) => {
-        return Observable.of(response);
```

Comment:
Fun and interesting Angular/Typescript shenanigans... This code already returns an Observable. Wrapping it in an `Observable.of` creates a nested Observable.

Reply:
Hmmm I specifically remember being mad about this and then getting a type error when I removed it...